Automattic/newspack-theme
# feat: add Flipboard icon for the social links menu

### All Submissions:


### Changes proposed in this Pull Request:

This PR adds a social links icon for Flipboard.

Closes #1169

### How to test the changes in this Pull Request:

1. Set up a social links menu; include a link for Flipboard (eg. https://flipboard.com/@VoiceofOC)
2. View on the front end; note that the Flipboard icon is using the fallback 'link' icon:

![image](https://user-images.githubusercontent.com/177561/101698455-5857fc80-3a2e-11eb-920e-79618f91b52d.png)

3. Apply the PR and run `npm run build`.
4. Confirm that the Flipboard link now uses the correct icon, and that it seems to be scaled well compared to your other icons:

![image](https://user-images.githubusercontent.com/177561/101698427-4c6c3a80-3a2e-11eb-9886-5a806ef152e7.png)

### Other information:
